RecoTracker/ConversionSeedGenerators/python/PhotonConversionTrajectorySeedProducerFromSingleLeg_cfi.py

```diff
@@ -23,7 +23,7 @@
                                                   ClusterCheckPSet = cms.PSet(
                                                       PixelClusterCollectionLabel = cms.InputTag("siPixelClusters"),
                                                       MaxNumberOfCosmicClusters = cms.uint32(400000),
-                                                      doClusterCheck = cms.bool(True),
+                                                      doClusterCheck = cms.bool(False),
```

Comment:
thanks for the good suggestion! I decided just to take the entire cluster charge cut from a central location. Not from there but for the one which most other seeds use
https://github.com/cms-sw/cmssw/blob/master/RecoTracker/TkSeedGenerator/python/SeedGeneratorFromRegionHitsEDProducer_cfi.py